.bazelrc

```diff
@@ -19,35 +19,28 @@ test:lint --test_tag_filters=lint
 # bazel test //... --config=unit
 test:unit --test_tag_filters=-lint
 
-# --config=remote-cache enables a remote bazel cache
-# Note needs an instance name
-# See --config=remote-fejta-cache for a concrete example
-build:remote-cache --remote_cache=remotebuildexecution.googleapis.com
-build:remote-cache --tls_enabled=true
-build:remote-cache --remote_timeout=3600
-build:remote-cache --auth_enabled=true
-
-# --config=remote adds remote execution to the --config=remote-cache
 # Note needs an instance name
 # See --config=remote-fejta for a concrete example
-build:remote --config=remote-cache
-build:remote --remote_executor=remotebuildexecution.googleapis.com
+# https://github.com/bazelbuild/bazel-toolchains/blob/master/bazelrc/bazel-0.27.0.bazelrc
```

Comment:
since this just matches the upstream, LGTM
however, noting that I did _not_ review all of these flags individually, I'm assuming the defaults changed around spawn strategy etc.

Reply:
Yeah they are rather mysterious. I'm just assuming the RBE team knows what they are doing